workflow.yml

```diff
@@ -0,0 +1,74 @@
+  contact:
+    name: GitHub Source Code
+    url: https://github.com/w3c-ccg/vc-api
+paths:
+  /workflows/{name}/start:
+    post:
+      summary: Start a workflow.
+      description:
+        Starts a workflow with the given name. Optionally, additional information can be provided to the server in the request body that can help it fine tune what information to request from the client in order to continue with the workflow.
+      requestBody:
+        description: Optional and arbitrary POST data that the client might provide to the server to help it fine tune what information to request from the client in order to continue with the workflow.
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - type: object
+                - type: string
+      responses:
+        "201":
+          description: A new workflow was successfully created for the client.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/WorkflowStartResponse"
+        "400":
+          description: The workflow request was malformed.
+        "500":
+          description: Internal Error
+        "501":
+          description: Not implemented
+  /workflows/{id}/presentations:
+    post:
+      summary: Present information to an active workflow.
+      operationId: storePresentation
+      description: "Delivery endpoint for a client-holder to provide a presentation in compliance with the Verifiable Presentation Request provided in the Notification call. This server-holder MUST match the 'domain' and 'challenge' to a 'domain' and 'challenge' previously sent in a Notification Response, and not already received. The server-holder MUST verify the Presentation. The server MUST store the Presentation if verification passes."
+      requestBody:
+        description: Details the client provides to the server to help it decide if the presentation should be made.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/VerifiablePresentation"
+      responses:
+        "200":
+          description: Presentation was successful
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VerifiablePresentation"
+        "201":
+          description: Presentation was received and a new workflow was successfully created for the client.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/WorkflowStartResponse"
+        "400":
+          description: Presentation is malformed
+        "501":
+          description: Not Implemented
+        "500":
+          description: Internal Server Error
+components:
+  schemas:
+    WorkflowStartResponse:
+      $ref: "./components/WorkflowStartResponse.yml#/components/schemas/WorkflowStartResponse"
```

Comment:
you can acctually seperate schemas and responses with folders, see https://github.com/dgarcia360/openapi-boilerplate/tree/master/src

Reply:
Sounds like something we should do. Will do this in a separate PR.